themes/default/layouts/partials/registry/package.html

```diff
@@ -66,13 +72,38 @@ <h3 class="text-gray-900 no-anchor">
                     {{ end }}
 
                     <!-- Fixed-height spacer accounts for the minimum space required by the tags. -->
-                    <div class="h-14"></div>
+                    <div class="spacer h-14"></div>
 
                     <!-- Tags are anchored to the bottom of the card. -->
                     <ul class="package-tags mt-2 absolute bottom-8">
                         <li class="package-tag-type" data-filter-group="type" data-filter-value="{{ $packageType }}">{{ $packageTypeName }}</li>
                         <li class="package-tag-category" data-filter-group="category" data-filter-value="{{ $packageCategory }}">{{ $packageCategoryName }}</li>
                     </ul>
+
+                    <div class="maintainer flex items-center flex-grow justify-end mt-2 absolute bottom-8 h-14">
+                        <div class="text-right">
+                            <pulumi-tooltip>
```

Comment:
Not a huge deal, but the tooltip seems to be floatin' off in space a bit:

![image](https://user-images.githubusercontent.com/274700/156422204-11a6fb2e-e7a3-4df5-95ad-c611cb381c5b.png)